### PR TITLE
Added RichTextAreaInput for ActionText

### PR DESCRIPTION
**Pull request by itsterry**
For #1638 I've added the appropriate input. 

I've also added a test that passes, but I'm not familiar enough with the rich_text_area in Rails to know what else I should be testing for: feedback appreciated!

I've tested this on a project in development and it seems to work nicely, but I haven't tried adding any optional attributes, so might be worth exploring

### CODE REVIEW

**Comment by yfxie** [comment]
@itsterry Thanks for your contribution. It works well. However, be careful when Bootsrap theme is applied.
`.form-control` defines a static height and the Trix editor will inherit the class.
This issue not happen for text input because of `textarea.form-control { height: auto; }`
It should better to define a new wrapper for Bootstrap theme.

**Comment by itsterry** [comment]
Thank you @yfxie - if it's cool with you, I'll leave that to someone with CSS smarts. I'm a Back End kind of guy :)

File: test/inputs/rich_text_area_input_test.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+# encoding: UTF-8
```
Comment (feliperenan) [inline test/inputs/rich_text_area_input_test.rb]: I believe we don't need this :scissors:

**Comment by IlianaP** [comment]
Hello! I am having a hard time figuring out how to use the rich_text update. I read through a bunch of threads - tried 
`<%= f.input :body, as: :rich_text_area %>`
`<%= f.rich_text_area :body %>`

Also tried doing a separate autoload file, but it's not working - I keep getting the Rails undefined method `rich_text_area' for #<SimpleForm::FormBuilder:0x00007fb4fd496218> error. 

I am running Rails 5.0.7.2 and have installed the simple form gem as just simple_form. 

Could someone help me out?

**Comment by yfxie** [comment]
> Hello! I am having a hard time figuring out how to use the rich_text update. I read through a bunch of threads - tried
> `<%= f.input :body, as: :rich_text_area %>`
> `<%= f.rich_text_area :body %>`
> 
> Also tried doing a separate autoload file, but it's not working - I keep getting the Rails undefined method `rich_text_area' for #SimpleForm::FormBuilder:0x00007fb4fd496218 error.
> 
> I am running Rails 5.0.7.2 and have installed the simple form gem as just simple_form.
> 
> Could someone help me out?

Rails introduced ActionText since version 6(see: https://edgeguides.rubyonrails.org/6_0_release_notes.html).
The method `rich_text_area` can be found here(https://apidock.com/rails/ActionView/Helpers/FormHelper/rich_text_area).

**Comment by itsterry** [comment]
Hey @IlianaP 

We had this problem before and solved it in a fork of this project, which you're welcome to use. It's at https://github.com/morsedigital/simple_form

Hope it helps!

**Pull request merged**